### ISSUE TITLE
Error in email validation with internationalized domain name and national alphabets

### ISSUE THREAD

**Issue by Quazer** [Semantic-Org/Semantic-UI#3814]
We can use national symbols in email (internationalized domain name via ICAAN). Email validations of forms fields is wrong. Example: [http://jsfiddle.net/Quazer/axLgoy8v/2/](http://jsfiddle.net/Quazer/axLgoy8v/2/).

Wrong code in **src\definitions\behaviors\form.js**
 ```
regExp: {
...
    email   : "[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?",
...
  },
```
For check the validity of the Russian-language addresses I use the following crutch:
```
<script type="text/javascript">
    $.fn.form.settings.regExp.email = "[a-z0-9а-я!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9а-я!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9а-я](?:[a-z0-9а-я-]*[a-z0-9а-я])?\\.)+[a-z0-9а-я](?:[a-z0-9а-я-]*[a-z0-9а-я])?";
</script>
```

I propose to make the correction.

**Comment by avalanche1** [comment]
I would propose to do the following: `/.+@.+\..+/i`
https://davidcel.is/posts/stop-validating-email-addresses-with-regex/

**Comment by jlukic** [comment]
Fixed in #3955 with a reasonable email RegExp from @kevinresol. If there's any that don't pass that filter feel free to ping me.
